.docs/content/2.guide/1.how-to/how-to-configure-partitions.md

```diff
@@ -153,7 +148,7 @@ TaskOptions = new TaskOptions{
 
 Props = new Properties(
     TaskOptions,
-    configuration.GetSection("Grpc")["EndPoint"],
+    configuration.GetSection("gRPC")["EndPoint"],
```

Comment:
Well, that one is tricky. I would check the code. I'm not sure that it will work like what you proposed

Reply:
Yep, the case-policy make an error here. Will check if we can disable it for the line.